hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/secure/TestRouterDelegationTokenSecretManager.java

```diff
@@ -1,201 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
have moved this class to router.security package rather than router.secure package & refactored tests to test the public methods of router secret manager rather than testing it internally. 

these test cases, validate the database storing / retrieval logic internally and this makes the test cases independent of the database implementations..